deploy-manage/upgrade/deployment-or-cluster/upgrade-on-ech.md

```diff
@@ -16,10 +16,12 @@ products:
 
 Once you are [prepared to upgrade](/deploy-manage/upgrade/prepare-to-upgrade.md), a single click in the {{ecloud}} console can upgrade a deployment to a newer version, add more processing capacity, change plugins, and enable or disable high availability, all at the same time. During the upgrade process, {{es}}, {{kib}}, Elastic APM, and all of your deployment components are upgraded simultaneously.
 
-You can perform minor version upgrades, upgrades from 8.18 to 9.0.0, and cluster configuration changes with no downtime. {{ecloud}} only supports upgrades to released versions. Release candidate builds and master snapshots are not supported.
+For HA (high availability) deployments, you can perform minor version upgrades, upgrades from 8.18 to 9.0.0, and cluster configuration changes with no downtime. {{ecloud}} only supports upgrades to released versions. Release candidate builds and master snapshots are not supported.
 
 ::::{important}
 Although it’s simple to upgrade an {{ecloud}} deployment, the new version might include breaking changes that affect your application. Ensure you review breaking changes and deprecation logs, make any necessary changes, and test against the new version before upgrading your production deployment.
+
+During the Kibana instances upgrade, Kibana is inaccessible.
```

Comment:
I don't think this notice should be in this same `important` block because the two subjects aren't really related. I'd do this as an additional paragraph instead - see my other comment